### ISSUE TITLE
slide 14 typo

### ISSUE THREAD

**Issue by pachadotdev** [jenniferthompson/RLadiesIntroToPurrr#3]
me again, slide 14 says

```r
map(datalist, ~ lm(value = year * region, data = .))
``` 

it should be

```r
map(datalist, ~ lm(value ~ year * region, data = .))
```

otherwise the result is

```
Error in `contrasts<-`(`*tmp*`, value = contr.funs[1 + isOF[nn]]) : 
  contrasts can be applied only to factors with 2 or more levels
In addition: Warning message:
In storage.mode(v) <- "double" : NAs introduced by coercion
```

thanks for the amazing tutorial !!!

**Comment by jenniferthompson** [comment]
Ha - the down side of using Keynote instead of RMarkdown for slides is that it doesn't catch these typos :) The Rmd should be right. Thanks so much for the heads up! I'll fix it in the next version!

**Comment by jenniferthompson** [comment]
Finally fixed the typo 6c83e45031e094aa07051cbd3dacb3d55d844b7b - thanks again @pachamaltese!
